bottomnavigation/bottomnavigation/src/main/java/com/microsoft/device/dualscreen/bottomnavigation/BottomNavigationView.kt

```diff
@@ -226,18 +276,24 @@ open class SurfaceDuoBottomNavigationView @JvmOverloads constructor(
 
     private fun BottomNavigationMenuView.arrangeOnScreen(firstBtnIndex: Int, secondBtnIndex: Int) {
         if (width != totalScreenWidth) {
-            layout(0, 0, totalScreenWidth, (parent as SurfaceDuoBottomNavigationView).height)
+            layout(
+                0,
+                0,
+                totalScreenWidth,
+                (parent as com.microsoft.device.dualscreen.bottomnavigation.BottomNavigationView).height
```

Comment:
maybe better to create an alias of the whole package? so the line is shorter and if the package it's used later can also benefit of that.